examples/pytorch/README.md

```diff
@@ -252,3 +252,65 @@ or if in a Conda environment:
 ```bash
 conda install -c comet_ml -c anaconda -c conda-forge comet_ml
 ```
+
+### Neptune
+
+1. Install the Neptune client library:
+
+    pip:
+
+    ```bash
+    pip install neptune-client
+    ```
+
+    conda:
+
+    ```bash
+    conda install -c conda-forge neptune-client
+    ```
+
+1. Import `NeptuneCallback`:
+
+    ```python
+    from transformers.integrations import NeptuneCallback
+    ```
+
+1. Enable Neptune logging in your script:
+
+    - Create a Neptune callback and pass it to the Trainer:
+
+        ```python
+        callback = NeptuneCallback()
+        trainer = Trainer(
+            model,
+            ...
+            callbacks=[callback],
+        )
+        ```
+
+    - Alternatively, in your `TrainingArguments`, set the `report_to` argument to `"neptune"`:
+
+        ```python
+        training_args = TrainingArguments(
+            "quick-training-distilbert-mrpc", 
+            evaluation_strategy="steps",
+            eval_steps = 20,
+            report_to = "neptune",
+        )
+        trainer = Trainer(
+            model,
+            training_args,
+            ...
+        )
+        ```
+
+1. When you start the training with `trainer.train()`, your metadata will be logged in Neptune.
+
+You can pass your **Neptune API token** and **project name** when creating the callback, but the recommended way is to save them as environment variables:
+
+| Environment variable | Value                                                |
+| :------------------- | :--------------------------------------------------- |
+| `NEPTUNE_API_TOKEN`  | Your Neptune API token. To find and copy it, click your Neptune avatar and select **Get your API token**. |
+| `NEPTUNE_PROJECT` | The full name of your Neptune project (`workspace-name/project-name`). To find and copy it, head to **project settings** &rarr; **Properties**. |
+
+For detailed instructions and examples, see the [Neptune docs](https://docs.neptune.ai).
```

Comment:
It should be to HF docs guide. HF users who start on this page, are interested in the integration.